mobile/library/common/config/config.cc

```diff
@@ -498,6 +498,8 @@ stats_sinks: *stats_sinks
       patterns:
         - prefix: cluster.base.
         - prefix: cluster.base_h2.
+        - prefix: sds.
+        - prefix: runtime.
```

Comment:
Nice!

Reply:
How many stats does this effectively enable - as in, how many more are tracked by Envoy after that change (we can just look at the number of active stats after we open the example app I guess)? (also, you may need to rebase as `patterns:` were updated in main)